### DISCUSSION TITLE
Where can I find the non-energy input of heat pump in GCAM-USA?

### DISCUSSION THREAD

**Discussion by huilinya** [JGCRI/gcam-core#244] [Modifying GCAM]
Hi! I found the Building_USA.xml, but in there the heat pumps only have minicam-energy-input. What about non-energy input (capital cost)? Is there a way to mode the change of non-energy input? Thank you!

**Comment by pkyle** [comment]
The non-energy costs are in the `global-technology-database` portion of the file, as they're currently assumed to be the same in all 51 states. Those in turn are read to the XML from the data in the table `inst/extdata/gcam-usa/A44.globaltech_cost.csv`. In that table, the costs are indicated in 1975$/GJ of useful energy, which is calculated in an off-line spreadsheet from assumptions about capital costs, O&M costs, equipment lifetimes, and discount rates, as well as the expected annual output for a typical unit. So it's a bit abstracted from market data that you could look up in the literature, but at the same time you can easily represent the impacts of e.g. reducing the costs of a technology by some percentage. The vast majority of the non-energy costs of all of these technologies is the capital cost; non-fuel O&M tends to be pretty low (in the real world that represents repairs and maintenance on the equipment).

**Reply by huilinya** [reply]
Thank you!!
